README.adoc

```diff
@@ -131,8 +140,11 @@ The `@Counted` annotation applies to the `list()` method. This annotation counts
 inventory.
 * The `absolute` field makes the name of this annotation take the name of the method.
 * The `description` field is optional and describes the purpose of the metric annotation.
-* The default value of the `monotonic` field is `false`. When `false`,
-the annotation counts the current invocations of the annotated method, and the counter is incremented before the annotated method returns. However, set the value to `true` instead.
+* The default value of the `monotonic` field is `false` which means that the counter is incremented
+ while the annotated method is invoked and decremented after the annotated method returns. However, set the value to `true` to only increment the counter.
```

Comment:
Please add a comma after "false" and before "which."

Also, change the second sentence to, "However, set the value to `true` to increment only the counter."

Reply:
@sishida ok thanks